src/status_im/contexts/wallet/events.cljs

```diff
@@ -386,3 +386,35 @@
             {:title   title
              :subject title
              :message content})]]}))
+
+(defn store-secret-phrase
```

Comment:
these events file is growing too large, we should split this file into more categories.

i.e we have an \
effects/account_generation
effects/account
etc etc 👍 

but we can do as a follow up.